### ISSUE TITLE
Convert all source code to maven project

### ISSUE THREAD

**Issue by Krzysztof1985** [habuma/spring-in-action-4-samples#5]
Hi!

I have bought your book from manning.com but source code is horrible!
Please could you convert whole Spring in Action 4th edition to maven project please?
It's easier and simplier than using gradle....

Regards
Krzysztof

**Comment by topriddy** [comment]
Yes please. I'm actually considering forking the project and doing just this.

**Comment by vijayrajagopal996** [comment]
Please do convert this whole project as a maven

**Comment by glocknergrat** [comment]
To convert the examples from the book to eclipse projects  (on a Linux OS) do:
./gradlew eclipse in the folder of the project.

**Comment by yawak** [comment]
I fork  this project and converting it to maven. I think they just need to add maven plugin and a group for pom to be generated automatically.

Example:
apply plugin: 'maven'
group = 'com.sia'

Reference:
https://codexplo.wordpress.com/2014/07/20/gradle-to-maven-conversion-and-vice-versa/

**Comment by jdzhang1221** [comment]
> I fork this project and converting it to maven. I think they just need to add maven plugin and a group for pom to be generated automatically.
> 
> Example:
> apply plugin: 'maven'
> group = 'com.sia'
> 
> Reference:
> https://codexplo.wordpress.com/2014/07/20/gradle-to-maven-conversion-and-vice-versa/

Are there any plans to convert to maven projects?

**Comment by yawak** [comment]
> 
> 
> > I fork this project and converting it to maven. I think they just need to add maven plugin and a group for pom to be generated automatically.
> > Example:
> > apply plugin: 'maven'
> > group = 'com.sia'
> > Reference:
> > https://codexplo.wordpress.com/2014/07/20/gradle-to-maven-conversion-and-vice-versa/
> 
> Are there any plans to convert to maven projects?

See: https://github.com/yawak/SpringInActionExamples

Planning to do a pull request in the future :)
